dashboard/app/views/devise/shared/_links.haml

```diff
@@ -1,38 +1,28 @@
-- if controller_name != 'sessions'
+-# Unless we're already on the Sign in page, show a Sign in link
+- unless controller_name == 'sessions'
   = t('auth.already_signedup')
   = link_to t('nav.user.signin'), new_session_path(resource_name)
   %br/
   %br/
-- if devise_mapping.registerable? && controller_name != 'registrations'
-  = t('auth.notyet_signedup')
-  = link_to t('nav.user.signup'), new_user_registration_path
-  %br/
-  %br/
--# If we are in sign-in (but not sign-up) and can offer to recover forgotten password.
-- if controller_name == 'sessions' && devise_mapping.recoverable? && controller_name != 'passwords'
-  = link_to t('auth.forgot_password'), new_password_path(resource_name)
```

Comment:
Now the only time we show forgot-password link is done inline in the sign-in page.